docs/examples/audio/spectrogram.ipynb

```diff
@@ -0,0 +1,272 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background\n",
+    "\n",
+    "In this example we will go through the steps to build a DALI audio processing pipeline, including the calculation of a spectrogram. A spectrogram is a representation of a signal (e.g. an audio signal) that shows the evolution of the spectrum of frequencies on the time domain.\n",
+    "\n",
+    "Typically, and in our DALI implementation, a spectrogram is calculated by computing the fast fourier transform (FFT) over a series of overlapping windows extracted from the original signal. The process of dividing the signal in short term sequences of fixed size and applying FFT on those independently is called Short-time Fourier transform (STFT). The spectrogram is then calculated as the power of the complex magnitude of the STFT.\n",
+    "\n",
+    "Extracting short term windows of the original image affects the calculated spectrum by producing aliasing artifacts. This is often call spectral leakage. To control/reduce the spectral leakage effect, we use different window functions when extracting short term windows. Some examples of window functions are: Hann, rectangular, Hanning.\n",
+    "\n",
+    "It is beyond the scope of this example to go deeper into the details of the signal processing concepts we mentioned above."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reference implementation\n",
+    "\n",
+    "To verify the correctness of DALI's implementation, we will compare against librosa (https://librosa.github.io/librosa/) implementation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python2.7/dist-packages/librosa/__init__.py:35: FutureWarning: You are using librosa with Python 2. Please note that librosa 0.7 will be the last version to support Python 2, after which it will require Python 3 or later.\n",
+      "  FutureWarning)\n",
+      "/usr/local/lib/python2.7/dist-packages/librosa/core/audio.py:146: UserWarning: PySoundFile failed. Trying audioread instead.\n",
+      "  warnings.warn('PySoundFile failed. Trying audioread instead.')\n"
+     ]
+    }
+   ],
+   "source": [
+    "import librosa as librosa\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline  \n",
+    "import librosa.display\n",
+    "\n",
+    "y, sr = librosa.load(librosa.util.example_audio_file())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Librosa provides an API to calculate the STFT, producing a complex output (i.e. complex numbers). It is then trivial to calculate the power spectrum from the complex STFT by the following"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n_fft=2048 \n",
```

Comment:
I would describe what that arguments mean.

Reply:
Will do